app/components/donations/create-donation.js

```diff
@@ -1,11 +1,24 @@
 import Ember from 'ember';
 
-const { Component, computed, isEmpty } = Ember;
+const { Component, computed, get, isEmpty, set } = Ember;
 
 export default Component.extend({
   classNames: ['create-donation'],
-  presetAmounts: [10, 15, 25, 50],
+
   amount: 10,
+  customAmount: null,
+  presetAmounts: [10, 15, 25, 50],
+
+  init() {
+    this._super(...arguments);
+
+    let amount = get(this, 'amount');
+    let presetAmounts = get(this, 'presetAmounts');
+
+    if (!presetAmounts.includes(amount)) {
+      set(this, 'customAmount', amount);
+    }
```

Comment:
Amount handling here is a little funky for floats, etc. I would also expect the URL to actually change when the `customAmount` changes, too.